CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 * [#2665](https://github.com/thanos-io/thanos/pull/2665) Swift: fix issue with missing Content-Type HTTP headers.
 - [#2800](https://github.com/thanos-io/thanos/pull/2800) Query: Fix handling of `--web.external-prefix` and `--web.route-prefix`
 - [#2834](https://github.com/thanos-io/thanos/pull/2834) Query: Fix rendered JSON state value for rules and alerts should be in lowercase
+- [#2848](https://github.com/thanos-io/thanos/pull/2848) Query: Fix remove erroneous error when viewing stores.
```

Comment:
We don't need this entry. Since this is a fix for a change that hasn't merged yet.